hal/tpm_io_infineon.c

```diff
@@ -51,7 +52,6 @@
         #define TPM2_I2C_ADDR 0x2e
     #endif
 
-    #if defined(CY_USING_HAL)
```

Comment:
Please don't remove this code or the code below at 149-152. Thank you

Reply:
I had to read that a few times to understand what's going on. 

Does that mean no I2C available / implemented on Infineon Tricore?